config/routes.rb

```diff
@@ -60,8 +60,15 @@
       post "review", action: :create
       get "success", action: :success
       get "not-eligible", action: :not_eligible
-      get "already-nominated", action: :already_nominated
+      get "already-renominated", action: :already_renominated
```

Comment:
Why do we need to change this? Will this not break what it was being used for before?

Reply:
I've changed the method names - It's because the description does not match with the flow - and it clashes with nomination flow - all tests are working plus there is a whole feature file testing it